content/docs/2.13/concepts/authentication.md

```diff
@@ -274,11 +276,13 @@ Currently we support the following:
 
 ```yaml
 podIdentity:
-  provider: none | azure | azure-workload | aws-eks | aws-kiam  # Optional. Default: none
-  identityId: <identity-id>                                     # Optional. Only used by azure & azure-workload providers.
+  provider: none | azure | azure-workload | aws | aws-eks | aws-kiam  # Optional. Default: none
+  identityId: <identity-id>                                           # Optional. Only used by azure & azure-workload providers.
+  roleArn: <role-arn>                                                 # Optional. Only used by aws provider.
+  identityOwner: keda|workload                                        # Optional. Only used by aws provider.
 ```
 
-#### Azure Pod Identity
+#### ([DEPRECATED](https://github.com/kedacore/keda/discussions/5362)) Azure Pod Identity
```

Comment:
Let's add a section inside here, not in the title because it's too aggressive + breaks existing links.

Also, as discussed in standup we'll get rid of these sections soon :)

Reply:
Umm... The renaming breaks the links too, so maybe I should revert the aws renaming.
Wdyt?

I'll move the deprecation to a quote in the first line of each auth

Reply:
I think for the AWS ones its ok, but I would not change it for deprecation notice